tests/fixtures/base-openapi.yml

```diff
@@ -22,15 +22,32 @@ paths:
       description: get an example
       operationId: get-example
       tags: [foo]
+      parameters:
+        - in: query
+          name: query_param_1
+          schema:
+            type: integer
+        - $ref: '#/components/parameters/QueryParamComponent'
+        - in: header
+          name: My-Header
+          schema:
+            type: integer
+        - $ref: '#/components/parameters/HeaderParamComponent'
       responses:
         '200':
           description: ok
           content:
             application/json:
               schema:
                 $ref: '#/components/schemas/Example'
-        '500':
-          description: server error
+        '404':
+          description: not found
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+        default:
```

Comment:
What is with other http codes? Will they be added later?

Reply:
http codes are checked by https://opensource.zalando.com/restful-api-guidelines/#150
will add that later